### DISCUSSION TITLE
wifi issue on macbook pro a1278 (2011)

### DISCUSSION THREAD

**Discussion by G14Nnl** [elementary/triage#228] [Q&A]
although i installed bcmwl kernel source on my macbook pro, can't get wifi working in any way. i've tried some guides found by googling but none of them have worked. how could i solve this issue, can someone help?

**Comment by mpanhans** [comment]
Have you verified that the wifi on that machine works on MacOS? Just want to make sure it is a linux issue and not a hardware issue.
If you type `lspci` into the Terminal, what model number does it give under network controller?
